### DISCUSSION TITLE
Provide reasoning for striking down server-affecting suggestions.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5864] [Suggestions]
With the [recent news about mythic dungeons](https://www.chromiecraft.com/en/mythic-dungeons-and-worldbosses/), a certain sentence was written that struck me.

```
There was helpful input from many other staffers and players through their guild representatives and I’m grateful for all the feedback and points you have raised, even if we decided against adding them for one reason or another.
```

I can speak from personal experience that my guild gave feedback and certain points were unanimous, though it seems this feedback was not accepted. The specific piece of feedback I'm thinking of is not really important here, but in the future, I personally would greatly appreciate if some sort of feedback section or page was made where the various commonly seen points of discussion were pointed out and given reasoning as to why they weren't accepted or implemented in the way they were suggested.

Up to this point, every time a controversial change is made, these common points are brought up and either no defense at all or an (in my opinion of course) insufficient defense is said. This happens multiple times over the course of however long, and I feel having a page *with* the announcement that can be linked whenever these points are brought up on a controversial decision would save time for staff and help resolve tension for those who feel their feedback was not heard or discarded without a thought.

**Comment by Aethereus-ManaOnion** [comment]
100%

**Comment by balleny** [comment]
I would also agree, and at the same time point at this past proposal for more transparency
https://github.com/chromiecraft/chromiecraft/discussions/4916 (Making these discord guild representative channels read only for non-representative people - or more everything into github instead)
